articles/140_topic_and_service_name_mapping.md

```diff
@@ -96,7 +94,6 @@ Fully qualified names have these additional restrictions:
 - must not contain tilde (`~`) or curly braces (`{}`)
 
 Note that expanded substitutions must result in a valid name.
-For example, a name `foo_{bar}` is not valid if `bar` is expanded as `_baz`, because that would result in a fully qualified name of `foo__baz` which contains repeated underscores (`_`).
```

Comment:
There are other things that aren't allowed, for example `/foo/{sub}/bar` would be invalid if `{sub}` were replaced with an empty string resulting int `/foo//bar`. Or if `{sub}` were replaced with `invalid!characters` which would result in `/foo/invalid!characters/bar`.